src/Microsoft.AspNetCore.Rewrite/Internal/IISUrlRewrite/UrlRewriteFileParser.cs

```diff
@@ -59,6 +64,10 @@ private void ParseRuleAttributes(XElement rule, UrlRewriteRuleBuilder builder)
             {
                 builder.Enabled = true;
             }
+            else
+            {
+                builder.Enabled = enabled;
+            }
```

Comment:
That's a good idea. The only thing is that if we don't parse the rest of the rule we allow the user to enter invalid rules since the checking isn't happening. Not a huge deal since the rule won't be enabled bit still something to consider